src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Solution/FinalizeSolutionStep.cs

```diff
@@ -29,6 +29,7 @@ public FinalizeSolutionStep(ILogger<FinalizeSolutionStep> logger)
 
         protected override Task<UpgradeStepApplyResult> ApplyImplAsync(IUpgradeContext context, CancellationToken token)
         {
+            // Finalizing will remove any progress tracked by Upgrade Assistant.
```

Comment:
Sorry - I meant to surface it to the user - probably in the description from the initialize method. We want it to be clear to them that if they select to finalize the solution, it will remove anything tracked by Upgrade Assistant

Reply:
Ah, ok - will do.